src/backend/aspen/api/deps.py

```diff
@@ -1,32 +1,18 @@
-from contextvars import ContextVar
-from typing import Optional
+from typing import AsyncGenerator
 
 from sqlalchemy.ext.asyncio import AsyncSession
+from starlette.requests import Request
 
 from aspen.api.settings import get_settings
 from aspen.database.connection import init_async_db
 
-session_context_var: ContextVar[Optional[AsyncSession]] = ContextVar(
-    "_session", default=None
-)
```

Comment:
This stuff was basically creating a global -- it was a means to an end to get the first version of this app working, but (no surprise) it made testing difficult. So we're now using a context manager to instantiate db sessions and letting fastapi manage dependency de-duplication.